homeassistant/components/just_nimbus/entity.py

```diff
@@ -0,0 +1,51 @@
+"""Base Entity for JustNimbus sensors."""
+from __future__ import annotations
+
+import justnimbus
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.helpers import update_coordinator
+from homeassistant.helpers.entity import DeviceInfo
+
+from . import JustNimbusCoordinator
+from .const import DOMAIN
+
+
+class JustNimbusEntity(
+    update_coordinator.CoordinatorEntity[justnimbus.JustNimbusModel],
+    SensorEntity,
+):
+    """Defines a base JustNimbus entity."""
+
+    def __init__(
+        self,
+        *,
+        client: justnimbus.JustNimbusClient,
+        entry_id: str,
```

Comment:
We don't have to pass the entry_id. The config entry is already available via the coordinator (`self.coordinator.config_entry.entry_id`).